.github/workflows/build.yaml

```
@@ -106,6 +106,44 @@ jobs:
           name: go-binaries-build
           path: go-binaries-build.tgz
 
+  pre-build-go-binaries-rcd:
+    if: |
+      (github.event.action == 'labeled' && github.event.label.name == 'turbo-build') ||
+      (github.event.action != 'labeled' && contains(github.event.pull_request.labels.*.name, 'turbo-build'))
```

These should go away:
```suggestion
    if: |
      (github.event.action == 'labeled' && github.event.label.name == 'turbo-build') ||
      (github.event.action != 'labeled' && contains(github.event.pull_request.labels.*.name, 'turbo-build'))
```